pages/linux/namcap.md

```diff
@@ -0,0 +1,12 @@
+# namcap
+
+> A tool to check binary packages and source PKGBUILDs for common packaging mistakes, which can also be automatically enabled.
+> More information: <https://wiki.archlinux.org/title/Namcap>.
+
+- To run `namcap` on a file, where filename is `PKGBUILD` or the name of a binary pkg.tar.xz:
```

Comment:
I think is best to split this example in 2: the first for `PKGBUILD` and another for packages.